--- src/org/cocos2dx/lib/Cocos2dxActivity.java	2015-02-26 22:17:46.000000000 -0800
+++ src/org/cocos2dx/lib/Cocos2dxActivity.java	2015-08-02 13:55:05.000000000 -0700
@@ -46,4 +46,7 @@
 import android.widget.FrameLayout;
+import android.content.Intent;
+import com.sdkbox.plugin.SDKBox;
+
 public abstract class Cocos2dxActivity extends Activity implements Cocos2dxHelperListener {
     // ===========================================================
     // Constants
@@ -220,4 +223,5 @@
         onLoadNativeLibraries();
+        SDKBox.init(this);
         sContext = this;
         this.mHandler = new Cocos2dxHandler(this);

@@ -248,10 +252,23 @@
     // ===========================================================
     // Methods for/from SuperClass/Interfaces
     // ===========================================================
-
+
+    @Override
+    protected void onStart() {
+        super.onStart();
+        SDKBox.onStart();
+    }
+
+    @Override
+    protected void onStop() {
+        super.onStop();
+        SDKBox.onStop();
+    }
+
     @Override
     protected void onResume() {
         super.onResume();
+        SDKBox.onResume();

         Cocos2dxHelper.onResume();
         this.mGLSurfaceView.onResume();
@@ -260,7 +277,8 @@
     @Override
     protected void onPause() {
         super.onPause();
-
+        SDKBox.onPause();
+
         Cocos2dxHelper.onPause();
         this.mGLSurfaceView.onPause();
     }
@@ -271,6 +289,13 @@
     }

     @Override
+    public void onBackPressed() {
+        if(!SDKBox.onBackPressed()) {
+            super.onBackPressed();
+        }
+    }
+
+    @Override
     public void showDialog(final String pTitle, final String pMessage) {
         Message msg = new Message();
         msg.what = Cocos2dxHandler.HANDLER_SHOW_DIALOG;
@@ -298,7 +323,9 @@
             listener.onActivityResult(requestCode, resultCode, data);
         }

-        super.onActivityResult(requestCode, resultCode, data);
+        if(!SDKBox.onActivityResult(requestCode, resultCode, data)) {
+            super.onActivityResult(requestCode, resultCode, data);
+        }
     }
