--- src/org/cocos2dx/lib/Cocos2dxActivity.java	2015-09-17 10:02:30.000000000 -0700
+++ src/org/cocos2dx/lib/Cocos2dxActivity.java	2015-09-17 10:07:59.000000000 -0700
@@ -50,6 +50,7 @@
 import javax.microedition.khronos.egl.EGLConfig;
 import javax.microedition.khronos.egl.EGLDisplay;
 
+import com.sdkbox.plugin.SDKBox;
 public abstract class Cocos2dxActivity extends Activity implements Cocos2dxHelperListener {
     // ===========================================================
     // Constants
@@ -262,6 +263,7 @@
         this.hideVirtualButton();
 
         onLoadNativeLibraries();
+        SDKBox.init(this);
 
         sContext = this;
         this.mHandler = new Cocos2dxHandler(this);
@@ -301,10 +303,22 @@
     // ===========================================================
 
     @Override
+    protected void onStart() {
+          super.onStart();
+          SDKBox.onStart();
+    }
+    @Override
+    protected void onStop() {
+          super.onStop();
+          SDKBox.onStop();
+    }
+
+    @Override
     protected void onResume() {
     	Log.d(TAG, "onResume()");
         super.onResume();
         this.hideVirtualButton();
+        SDKBox.onResume();
        	resumeIfHasFocus();
     }
     
@@ -329,10 +343,18 @@
     protected void onPause() {
     	Log.d(TAG, "onPause()");
         super.onPause();
+        SDKBox.onPause();
         Cocos2dxHelper.onPause();
         mGLSurfaceView.onPause();
     }
-    
+
+    @Override
+    public void onBackPressed() {
+          if(!SDKBox.onBackPressed()) {
+            super.onBackPressed();
+          }
+    }
+
     @Override
     protected void onDestroy() {
         super.onDestroy();
@@ -358,7 +380,9 @@
             listener.onActivityResult(requestCode, resultCode, data);
         }
 
-        super.onActivityResult(requestCode, resultCode, data);
+        if(!SDKBox.onActivityResult(requestCode, resultCode, data)) {
+            super.onActivityResult(requestCode, resultCode, data);
+        }
     }
 
 
